pythonAPI/api.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
rename file. Not intuitive what is api?

Reply:
This file is the one in which api is created
what should I name it?

Reply:
I have approved, but rename this file. Say "admin_app.py" or "admin_service.py"